renting/setting-up-renterd/linux/debian.md

```diff
@@ -141,7 +153,9 @@ sudo systemctl start renterd
 
 ## Next Steps
 
-Now that you have `renterd` installed and running, you can start using it to store and retrieve data on the Sia network. You can access the web interface by navigating to `http://127.0.0.1:9980` in your web browser. If you installed `renterd` on a remote machine or a server, you will need to create an SSH tunnel to access the web interface.
+Now that you have `renterd` installed and running, you can start using it to store and retrieve data on the Sia network. You can access the web interface by navigating to [http://127.0.0.1:9980](http://127.0.0.1:9980) in your web browser. If you installed `renterd` on a remote machine or a server, you will need to create an SSH tunnel to access the web interface.
```

Comment:
`nit` you could mention using a reverse proxy like nginx or caddy to expose `renterd` through https